pkg/warmup/target.go

```diff
@@ -57,15 +60,16 @@ func NewTarget(httpClient whttp.Client, grpcClient grpc.Client, options TargetOp
 
 func (t Target) waitForReadinessProbe(done <-chan struct{}) error {
 
-	log.Printf("waiting for %s target readiness probe", t.options.ReadinessPath)
+	log.Printf("waiting for %d:%s target readiness probe", t.options.ReadinessPort, t.options.ReadinessPath)
 	for {
 		select {
 		case <-time.After(time.Duration(t.options.ReadinessTimeoutInSeconds) * time.Second):
 			return fmt.Errorf("target readiness proble: timeout %d seconds exceeded", t.options.ReadinessTimeoutInSeconds)
 		case <-done:
 			return errors.New("target readiness probe: received done signal")
 		default:
-			if err := t.httpClient.Request(http.MethodGet, t.options.ReadinessPath, nil, nil); err != nil {
+			// FIXME: just http here? no grpc?
```

Comment:
this bit here seems odd. this file is not http specific but there's only one branch for http (none for grpc)

Reply:
Mittens doesn't support gRPC health checks atm. Would you mind raising an issue and we can look into it?